CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Zerologon exploiter writes runtime artifacts to a secure temporary directory
   instead of $HOME. #1143
 - Authentication mechanism to use bcrypt on server side. #1139
+- Modified/added details around server_config changes to documentation. #1164
```

Comment:
I don't think this is necessary. The documentation changes are implied by the changelog entries representing the work we did.